[blog] Michał Dudak joins Material-UI (#26700)
mui-org/material-ui: docs/pages/blog/michal-dudak-joining.md

---
title: Michał Dudak joins Material-UI
description: We are excited to share that Michał Dudak has joined Material-UI.

> date: 2021-06-11T00:00:00.000Z

Author: We can publish next Monday.

```suggestion
date: 2021-06-14T00:00:00.000Z
```

authors: ['oliviertassinari']
card: true
---

We are excited to share that [Michał Dudak](https://twitter.com/michaldudak) has joined Material-UI!
He started a couple of days ago full-time and is part of the core components team.

> Before joining Material-UI, Michał worked as a full-stack engineer at Future Processing, a high-end development agency in Poland.

Reviewer: ```suggestion
Before joining Material-UI, Michał worked as a full-stack engineer and software architect at Future Processing, a high-end development agency in Poland.
```

He has been a professional practitioner in the field for over a decade.
He admires great designs and clever solutions.

> Writing CSS is a pleasure for him.

Reviewer: ```suggestion
Creating with TypeScript and CSS is a pleasure for him.
```

We were impressed by his technical challenge.

> He has leveraged most of the best practices we enforce when writing components without prior experience contributing to our codebase.

Reviewer 2: ```suggestion
He leveraged most of the best practices we enforce when writing components, without prior experience contributing to our codebase.
```

> While we can't predict the future, Michał is now responsible for the development of a new vertical for Material-UI: the unstyled & headless React components.

Reviewer 2: ```suggestion
While we can't predict the future, Michał is currently responsible for the development of a new vertical for Material-UI: the unstyled & headless React components.
```
?

> Marija has initiated this effort with a couple of new modules under the [@material-ui/unstyled](https://unpkg.com/browse/@material-ui/unstyled@next/) package.

Reviewer 2: ```suggestion
Marija initiated this effort with a couple of new modules under the [@material-ui/unstyled](https://unpkg.com/browse/@material-ui/unstyled@next/) package.
```

> This is a strategical effort for us with the intent to solve two problems:

Reviewer 2: ```suggestion
This is a strategic effort for us, with the intent to solve two problems:
```

> 1. Share logic between Material Design and the second different design specifications [Jun will work on](/blog/siriwat-kunaporn-joining/). This is important to stay efficient.

Reviewer 2: ```suggestion
1. Share logic between Material Design and the second design specification that [Jun will work on](/blog/siriwat-kunaporn-joining/). This is important to stay efficient.
```

> 2. Grow the audience of the library. It provides a new customization tradeoff. It maximizes freedom and a low bundle-size footprint. We have recently crossed the [20%](https://docs.google.com/spreadsheets/d/1l5j3Xjtvm9XZtmb4ulLiWElQaXSlZlyCWT5ONrQMpBo/edit#gid=0) market share threshold inside the React ecosystem (@material-ui/core downloads divided by react-dom downloads). We aim for [+20%](https://npm-stat.com/charts.html?package=@angular/core,@angular/material,@angular/cdk) of the market coming from this effort in the next four years.

Reviewer 2: ```suggestion
2. Grow the audience of the library. It provides a new customization tradeoff. It maximizes freedom and minimizes bundle-size. We have recently exceeded [20%](https://docs.google.com/spreadsheets/d/1l5j3Xjtvm9XZtmb4ulLiWElQaXSlZlyCWT5ONrQMpBo/edit#gid=0) market share of the React ecosystem (@material-ui/core downloads divided by react-dom downloads). We're aiming for [+20%](https://npm-stat.com/charts.html?package=@angular/core,@angular/material,@angular/cdk) of the market coming from this effort in the next four years.
```

Michał is the third of a series of five new positions we have recently opened and filled.

> We will reach 11 people at the end of June.

Reviewer 2: ```suggestion
We will reach 11 people by the end of June.
```
